interface/orders/orders_results.php

```diff
@@ -354,23 +354,23 @@ function validate(f) {
         $form_proc_type_desc = $ptrow['name'];
     }
 ?>
-    <?php xl('Procedure', 'e'); ?>:
+    <?php xlt('Procedure', 'e'); ?>:
    <input type='text' size='30' name='form_proc_type_desc'
     value='<?php echo addslashes($form_proc_type_desc) ?>'
     onclick='sel_proc_type()' onfocus='this.blur()'
-    title='<?php xl('Click to select the desired procedure', 'e'); ?>'
+    title='<?php xlt('Click to select the desired procedure', 'e'); ?>'
```

Comment:
We have 2 basic translation escaping function in the non javascript context, xlt() and xla().
xlt() is basically htmlspecialchars(xl(), ENT_NOQUOTES)
xla() is basically htmlspecialchars(xl(), ENT_QUOTES)
(as an aside, note the text() and attr() are shotcuts for above without the xl() translations calls)

xla is used within html attributes (hence the ENT_QUOTES flag, while xlt is used elsewhere). The above line is within html attribute (title attribute), so should use xla() here (also note removal of `e` parameter and adding of `echo`):
```
title='<?php echo xla('Click to select the desired procedure'); ?>'
```

Reply:
Thank you!!